examples/pytorch/example.py

```diff
@@ -19,7 +19,10 @@ class SHModule(torch.nn.Module):
     `torch.nn.Module`"""
 
     def __init__(self, l_max, normalized=False):
-        self.spherical_harmonics = sphericart.torch.SphericalHarmonics(l_max, normalized)
+        if normalized:
+            self.spherical_harmonics = sphericart.torch.SphericalHarmonics(l_max)
+        else:
+            self.spherical_harmonics = sphericart.torch.SolidHarmonics(l_max)
```

Comment:
Does it make sense to keep both options here now there there is no longer a normalized option? I would only demonstrate one of the calculator in the example and not muddy the water with more option than required

Reply:
Yes, let's just keep the SphericalHarmonics